data/json/items/generic.json

```
@@ -2401,6 +2401,21 @@
     "to_hit": -2,
     "flags": [ "TRADER_AVOID", "FIREWOOD" ]
   },
+  {
+    "type": "GENERIC",
+    "id": "pinecone_pig",
+    "name": { "str": "pinecone amulet" },
+    "category": "other",
+    "weight": "220 g",
+    "color": "brown",
+    "symbol": ",",
+    "description": "A pinecone with four small sticks shoved between its scales. An amulet of good luck and fortune, good thing it's so easy to make.",
+    "price": "10 cent",
+    "price_postapoc": "10 cent",
```

```suggestion
    "price_postapoc": "0 cent",
```

There's no way in hell people would pay for this